src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -559,6 +568,38 @@ The trusted code can store capabilities on the trusted stack when calling out of
 when returning to the same compartment.
 =====
 
+[#ycfg,reftext="ycfg"]
+===== CHERI Capability Encoding (ycfg)
+
+The <<ycfg>> register is used to identify which CHERI capability encoding is in use.
+The capability encoding determines both the in-memory representation of a CHERI capability
+and which CHERI features are available for software's use.
+The CSR exists in the _writable_ namespace and all bits are defined to be WARL,
+but, absent any future extensions, it is expected that each implementation has exactly one legal value.
+This CSR is always present in the machine mode CSR space;
+if the core supports user mode, a `uycfg` alias exists in the appropriate space,
+and, if the core supports supervisor mode, a `sycfg` alias exists, again in the appropriate space.
+
+The base specification defines the following values for <<ycfg>>:
+|===
+|Version value|Encoding
+|0|Reserved (unspecified capability encoding)
+|1|<<app_cap_description>> capabilities, with <<section_zylevels1>> features
+|2|<<app_cap_description>> capabilities, without <<section_zylevels1>> features
+|3|Reserved for CHERIoT's (RV32-only) encoding
```

Comment:
```suggestion
|0x01|<<app_cap_description>> capabilities, with <<section_zylevels1>> features
|0x11|<<app_cap_description>> capabilities, without <<section_zylevels1>> features
|0x02|Reserved for CHERIoT's (RV32-only) encoding
```
If we were to split into a major+minor scheme without having to add a new diagram for this register?

Reply:
I think that providing that adding the 0x10 value doesn't always have to mean levels then we're good to go - i.e. 0x1, 2, 3 in the first digit can have levels, and so adding 0x10 means levels. But maybe 0x4 in the first digit can never have levels, and so adding 0x10 in that case could mean something else (if that makes any sense).

So adding 0x10 to the "first option" which for the currently defined formats is "levels", it's not saying that adding 0x10 _always_ means levels.

Reply:
Yes exactly, that is my proposal. Having the second hex digit be 0x1 (i.e. 0x10) does not mean generally mean levels, only when combined with 0x1 in the least significant hex digit does it means levels (0x011). A hypothetical future change to the CHERIoT format would be 0x12 and the second hex digit would mean something completely different.